src/lib/network_pool/snark_pool.ml

```diff
@@ -484,6 +516,7 @@ let%test_module "random set test" =
 
     let config verifier =
       Mock_snark_pool.Resource_pool.make_config ~verifier ~trust_system
+        ~disk_location:"/tmp/snark-pool"
```

Comment:
I think this should probably go in `~/.coda-config` for now. We only store caches in `/tmp` at the moment; the rest of our persisted state is in `~/.coda-config` right now.

Reply:
This is for tests -- the actual path is in coda-config (it's set in coda.ml):

```ocaml
...
                  ; num_threads= snark_worker_parallelism_flag }
                ~snark_coordinator_key:run_snark_coordinator_flag
                ~snark_pool_disk_location:(conf_dir ^/ "snark_pool")
                ~wallets_disk_location:(conf_dir ^/ "wallets")
...
```